.htaccess

```diff
@@ -39,9 +39,8 @@ RewriteRule    ^admin/([a-z]+)/?$									index.php?s=admin&p=$1								[NC,L]
 
 #history section: specific objects
 RewriteRule		^history/buggy/([a-z0-9]+)/?$						index.php?s=history&p=buggy&urlkey=$1			[NC,L]
-RewriteRule		^history/org/([a-z]+)/?$							index.php?s=history&p=org&urlkey=$1				[NC,L]
-RewriteRule		^history/org/([0-9]+)/?$							index.php?s=history&p=org&id=$1					[NC,L]
-RewriteRule		^history/person/([0-9]+)/?$						index.php?s=history&p=person&id=$1				[NC,L]
+RewriteRule		^history/org/([a-z0-9]+)/?$							index.php?s=history&p=org&urlkey=$1				[NC,L]
```

Comment:
..did this only allow all numbers or all letters before?

Reply:
Yeah.  The new schema effectively made the choice to eliminate the urlkey fields that existed previously, and instead use the primary key as the urlkey.  This means in many (all?) cases our primary keys are now strings, but also human readable.  Some tables (person, heat) are a mix of numbers and more readable strings, but they are always stored as strings.

Since these are now the urlkeys for the specific-object pages, we need to support letters (we do keep it case-insensitive though), as well as, in one case, a dash.

Normally, this would make me a bit ill, but our data is so tiny I don't think this is a big deal in practice and it helps Jeremy with data entry and updating.  And, they're all indexed so any joins are going to be very fast anyway.